proto/api/v1/tenant.proto

```diff
@@ -125,7 +125,9 @@ message TenantMember {
   // Id is the user id of the member
   string id = 1;
   // Role is the role of the member
-  TenantRole role = 2;
+  TenantRole role = 2 [(validate.rules).enum = {
+    not_in: [0]
```

Comment:
sorry for complaining again, please add this validation to the new Service Request TenantServiceAddMemberRequest

Reply:
Better would be:

```
[(validate.rules).enum.defined_only = true];

```

Reply:
Okay will remove out of the API, the validation.
Only validation will be in the new admin function